### PR TITLE
Add timestamp tooltip

### LINKED ISSUE #598: Show exact date and time of a photo in sidebar
**Is your feature request related to a problem? Please describe.**
I want to see the exact date and time a photo was take/the file was created.

**Describe the solution you'd like**
I would like to see the timestamp in the sidebar.
image

Like this: `11.01.2019 20:38 (2 years ago)`.

**Describe alternatives you've considered**
Alternatively a tool tip when mouse over the approx. date would be fine.

Like here on  Github.
image

### PR DESCRIPTION
**Pull request by Pytal**
Closes nextcloud/photos#598

Related PR: nextcloud/nextcloud-vue#1971

### CODE REVIEW

File: apps/files/src/views/Sidebar.vue [hunk 1]
```diff
@@ -169,6 +169,14 @@ export default {
 			return OC.Util.relativeModifiedDate(this.fileInfo.mtime)
 		},
 
+		/**
+		 * File last modified full string
+		 * @returns {string}
+		 */
+		fullTime() {
+			return OC.Util.formatDate(this.fileInfo.mtime)
```
Comment (juliushaertl) [inline apps/files/src/views/Sidebar.vue:177]: We should switch to use the `@nextcloud/moment` package here as the formatDate global has beed deprecated:

`OC.Util.formatDate is deprecated and will be removed in Nextcloud 21. See @nextcloud/moment`

https://github.com/nextcloud/server/blob/3e624dc39966f54c6d418eee2e8141fc5c21f751/core/src/OC/util.js#L123

Comment (Pytal) [reply apps/files/src/views/Sidebar.vue]: I've made the changes with [@nextcloud/moment](https://nextcloud.github.io/nextcloud-moment/), what do you think?

**Review by skjnldsv — APPROVED** [review]
Code looks good to me :)
Needs an update of the vue components though

**Comment by Pytal** [comment]
bump :) just waiting on checks

**Review by juliusknorr — APPROVED** [review]

**Comment by juliushaertl** [comment]
@Pytal Needs another rebase and update of the bundles

**Comment by Pytal** [comment]
> @Pytal Needs another rebase and update of the bundles

Done and ready for launch 🚀

**Pull request merged**